src/main/java/com/google/sps/data/Player.java

```diff
@@ -13,6 +13,14 @@ public Player(String displayName, String email, String imageID) {
     this.imageID = imageID;
   }
 
+  public Player(String displayName, String email, String id, String imageID, String currentPageID) {
```

Comment:
Have this constructor call the other constructor, i.e.

this(displayName, email, imageId);
this.id = id;
this.currentPageID = currentPageID;